### DISCUSSION TITLE
Auth UML

### DISCUSSION THREAD

**Discussion by lxodud** [ASAP-Side/iOS-aljo#84] [Ideas]
<img width="870" alt="스크린샷 2024-04-01 오후 4 23 27" src="https://github.com/ASAP-Side/iOS-aljo/assets/85005933/54fb0045-c6e6-43de-8a06-4ea647bfc221">

[UML](https://lucid.app/lucidchart/565d9391-8bb6-4465-9f05-c2b05aeed3ab/edit?viewport_loc=-137%2C140%2C3166%2C1950%2C0_0&invitationId=inv_1451a54b-5400-497a-a924-c9e5e4fccd5f)

**Comment by lxodud** [comment]
@leegyoungmin

**Comment by leegyoungmin** [comment]
AuthManager 타입이 KakaoSDK, Apple Library를 사용하는 영역으로 추측됩니다. 해당 SDK들은 화면을 직접적으로 부르고, Redirect되는 부분이 있기때문에 View와 더 밀접한 관련이 있다고 생각됩니다. 즉, View가 직접적으로 의존하는 것이 더 자연스러운 흐름이라고 판단됩니다.
